_posts/en/newsletters/2025-02-07-newsletter.md

```diff
@@ -0,0 +1,580 @@
+  than other blocks in the same generation.  It's not possible to
+  achieve consensus on lateness, so ultimately DAGs with more
+  proof-of-work (PoW) are preferred over those with less PoW; the
+  developer of the DAA, Bob McElrath, [analyzed][mcelrath daa-latency]
+  the problem and a possible mitigation.
+
+  Pieter Wuille [commented][wuille prior] that the proposal appears
+  similar to a [2012 idea][miller stale] by Andrew Miller; Zawy
+  [agreed][zawy prior] and McElrath [will][mcelrath prior] update his
+  paper with a citation.  Sjors Provoost [discussed][provoost dag] the
+  complexity of handling DAG chains in Bitcoin Core's current
+  architecture, but noted that it might be easier using libbitcoin and
+  efficient using [utreexo][topic utreexo].  Zawy extensively
+  [simulated][zawy sim] the protocol and indicated that he was working
+  on additional simulations for variants of the protocol to find the
+  best mix of tradeoffs.
+
+  The last post to the discussion thread was made about a month before this
+  summary was written, but we expect Zawy and Braidpool developers are
+  continuing to analyze and implement the protocol.
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+- [BDK Wallet 1.1.0][] is a release of this library for building
+  Bitcoin-enabled applications.  It uses transaction version 2 by
+  default (improving privacy by allowing BDK transactions to blend in
+  with other wallets that must use v2 transactions due to their support
+  for relative locktimes, see [Newsletter #337][news337 bdk]).  It also
+  adds support for [compact block filters][topic compact block filters]
+  (see [Newsletter #339][news339 bdk-cpf]), in addition to "various bug
+  fixes and improvements".
+
+- [LND v0.18.5-beta.rc1][] is a release candidate for a minor version of
+  this popular LN node implementation.
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #21590][] implements a safegcd-based [modular
+  inversion][modularinversion] algorithm for MuHash3072 (see Newsletters
+  [#131][news131 muhash] and [#136][news136 gcd]), based on libsecp256k1’s implementation while adding
+  support for both 32-bit and 64-bit architectures and specializing for the
+  specific modulus. Benchmark results show an approximate 100× performance
```

Comment:
I would suggest dropping the two mentions of differences between the original libsecp256k1 implementation and the new one. The first is imprecise as it is worded and the second is a bit redundant:

  - libsecp256k1 had support for both archs, but it was split into two files, one for each arch; the change is that this code is now generic for both archs.
  - the specialization is for MuHash3072, but that is already mentioned.

```suggestion
  [#131][news131 muhash] and [#136][news136 gcd]), translated into C++ from libsecp256k1’s code. Benchmark results show an approximate 100× performance
```